dist/s_all

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
 
 # Run standard scripts.
+
 t=__wt.$$
 t_pfx=__s_all_tmp_
-trap 'rm -f $t *.pyc __tmp __wt.* __s_all_tmp*' 0 1 2 3 13 15
+allchildpids() { # Recursively get the pids of child processes.
+    ps -o pid,ppid | PID=$$ perl -naE '
```

Comment:
Do we have any perl experts on the team? This looks correct but I'd like an experienced pair of eyes on it.

Reply:
Ok let me port it to Python :)

Reply:
Thanks! The new code LGTM. Some small naming requests for readability:
`a` -> `children`
`(child_pid, parent_pid) = (pair[0], pair[1])`
`p` -> `descendants`